doc/source/train/examples/lightning/lightning_cola_advanced.ipynb

```diff
@@ -0,0 +1,465 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "(lightning_advanced_example)=\n",
+    "\n",
+    "# Finetune a BERT Text Classifier with LightningTrainer\n",
+    "\n",
+    ":::{note}\n",
+    "\n",
+    "This is an advanced example for {class}`LightningTrainer <ray.train.lightning.LightningTrainer>`, which demonstrates how to use LightningTrainer with `Ray Dataset` and `Batch Predictor`. \n",
```

Comment:
Recommendation linking to the documentation for the corresponding components:

https://docs.ray.io/en/master/data/dataset.html
https://docs.ray.io/en/master/ray-air/predictors.html